### PR TITLE
Get basic election & locality conversion working

### PR DESCRIPTION
**Pull request by cap10morgan**
We should go over what's going on in this PR together, including where the extension points are for generating the other CSV files.

### CODE REVIEW

**Comment by cap10morgan** [comment]
I also need to add some docstrings and tests, so I'll try to work on that today.

**Comment by cap10morgan** [comment]
@tank157 The README is on the master branch.

**Comment by tank157** [comment]
@cap10morgan Ah, sure enough it is, thanks!

File: src/vip_feed_format_converter/core.clj [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+(defn process [in-file out-dir]
+  (-> {:in-file in-file
+       :out-dir out-dir
+       :tag-path []}
+      open-input-file
+      set-handlers
+      election-headers
+      locality-headers
+      xml/parse-file
+      csv/write-files
```
Comment (tank157) [inline src/vip_feed_format_converter/core.clj]: Am I missing something here, Wes, or will the entire XML data structure be in memory (translated to a way we can write it out) by the time we get to the `csv/write-files` step? It seems like it, but I haven't taken the time to fully mentally run through the process and figure out how laziness might fit in.

Comment (cap10morgan) [reply src/vip_feed_format_converter/core.clj]: No, very little will be in memory. And it isn't due to laziness so much as the event-based parsing. Happy to hop onto a screenshare to walk through it.

**Comment by cap10morgan** [comment]
How does that look, @tank157?

**Pull request merged**